quickstarts/redisenterprise/dashboards/redisenterprisedashboards/active_active.json

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
@maguec , this will still not register. You will need to have the exact same dashboard name with only the numbers added.
current: `active_active_dashboard01`
expected: `active_active01`

Reply:
Sorry I left out the file tag. This is still not in the right format. Here is what the names will have to be changed to in order to register:
1. dash: `active_active.json` -> image: `active_active01.png`
2. dash: `cluster.json` -> image: `cluster01.png`
3. dash: `database.json` -> image: `database01.png`